db/migrate/20250522030006_create_report_mentions.rb

```diff
@@ -0,0 +1,12 @@
+class CreateReportMentions < ActiveRecord::Migration[7.0]
+  def change
+    create_table :report_mentions do |t|
```

Comment:
```suggestion
    create_table :mentions do |t|
```

現時点では日報以外で言及機能を作る予定はないのでreport_は省略していいかも
https://bootcamp.fjord.jp/pages/avoid-too-specific-var-name

Reply:
`report_`を削除し、その他必要な箇所を修正しました。